setup_interactive_session_on_VSC.md

```diff
@@ -0,0 +1,146 @@
+# Install OpenMM in an HPC environment for interactive use on VSC.
+
+Working with an HPC requires a basic knowledge of the Linux operating system. If you have never used Linux before, this will be rich learning experience. Still, it is worth the trouble, because in a computational  research lab, any serious simulation is performed on an HPC. Calculations on your laptop are only useful for teaching, preparing simulations or initial prototyping.
+
+1. Windows users that have not used Secure Shell (SSH) before, should first install a Secure Shell client. This is only needed on Windows, since MacOS and Linux come with a built-in `ssh` program. Th recommend SSH client for Windows is MobaXTerm (free version), which can be found here: https://mobaxterm.mobatek.net/
+
+
+2. You are now ready to request an HPC account, for which the exact procedure depends on your host institution. Students at Ghent University can [create a VSC account](request_vsc_account_ugent.md).
+
+3. Configure the SSH connection to the HPC.
+
+    - On Linux and MacOS, add the following to the file `~/.ssh/config`:
+
+        ```
+        Host hpc
+            Hostname login.hpc.UGent.be
+            user vsc4XXXX
+            IdentityFile /home/YYYY/.ssh/id_rsa
+        ```
+        where `XXXX` should be replaced by your VSC ID and `YYYY` is the username on your laptop.
+
+    - On Windows, ???
+
+4. Connect to the HPC
+
+    - On Linux and MacOS, run the following in a terminal:
+
+        ```bash
+        ssh hpc
+        ```
+
+    - On Windows, ???
+
+    Now that your account is working, go through the Linux and HPC tutorials here: https://www.ugent.be/hpc/en/support/documentation.htm. (These are somewhat geared to VSC users     at one of the Flemish universities, but they are general enough for audience.)
+
+5. After connecting to the HPC, determine a suitable location to install Minoconda. This is a slimmed-down version of Anaconda, which is suitable for non-local installations. The data directory should have sufficient quota (several Gigabytes) to contain the OpenMM installation. On the VSC clusters, this directory would be `$VSC_DATA`, but if you are working on another HPC, this might be different. In the instructions below, we will use `$VSC_DATA`, but this can be easily replaced.
+
+6. The installation of OpenMM takes several steps. First download and install Miniconda.
+
+   ```bash
+   cd $VSC_DATA
+   wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+   bash Miniconda3-latest-Linux-x86_64.sh -bfp ${VSC_DATA}/miniconda
+   vi ~/.bashrc
```

Comment:
I'd suggest changing this to `nano`. Exposing students to `vi` is a bit mean. :smile: